Library/Formula/gdb.rb

```diff
@@ -37,22 +51,29 @@ class Gdb < Formula
     depends_on UniversalBrewedPython
   end
 
+  if build.with? "brewed-python3"
+    depends_on UniversalBrewedPython3
+		patch :p1, :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!